src/Core/Components/NavMenu/FluentNavMenu.razor.js

```diff
@@ -22,6 +22,18 @@ export function onUpdate() {
     
 }
 
+export function onDispose() {
```

Comment:
When this `onDispose` method will be called?
I don't know how to detect when an HTML component will be "disposed".

Reply:
It will get called as part of the PageScript flow (not happening right now, but to be prepared for it)